Dockerfile

```diff
@@ -33,6 +33,9 @@ USER docsworker-xlarge
 
 WORKDIR /home/docsworker-xlarge
 
+# get shared.mk
+RUN curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/makefiles/shared.mk >> shared.mk
```

Comment:
A couple notes:
* `>>` works, but isn't necessary here: it'll *append* to the current file instead of overwriting it
* The `--output` option for curl is actually preferable over shell redirection: because file creation is handled by the shell, if curl fails, shell redirection will still end up creating an empty file.

Reply:
got it! I've changed it to curl <link> -o shared.mk